kiali-server/values.yaml

```diff
@@ -72,6 +72,7 @@ deployment:
     log_level: "info"
     time_field_format: "2006-01-02T15:04:05Z07:00"
     sampler_rate: "1"
+  extraEnvs: []
```

Comment:
For legacy reasons, all settings are snake_case, not camelCase. To remain consistent across all Kiali settings in the Kiali CR and server helm charts, use snake_case for any new settings.

Also to remain consistent with other analogous settings (such as existing settings of [`additional_labels`](https://github.com/kiali/helm-charts/blob/v1.84.0/kiali-server/values.yaml#L61) and [`additional_service_yaml`](https://github.com/kiali/helm-charts/blob/v1.84.0/kiali-server/values.yaml#L44)), extra_envs should be renamed `additional_envs`.